crates/sui-protocol-config/src/lib.rs

```diff
@@ -3720,6 +3721,18 @@ pub fn is_mysticeti_fpc_enabled_in_env() -> Option<bool> {
     None
 }
 
+#[inline(always)]
+pub fn in_antithesis() -> bool {
```

Comment:
Adding as dependency the `mysten_common` crate leads to a circular dependency . I am pretty sure there are better ways to avoid duplicating https://github.com/MystenLabs/sui/blob/c208ff2825d281f876d66a251b09e02f2cf65dfb/crates/mysten-common/src/lib.rs#L16 here (extracting in another common crate or just reuse this method in protocol config? ). Feel free to refactor.

Reply:
I think this has to be the way for now and we can refactor later. cc @mystenmark